tests/integration_tests/conftest.py

```diff
@@ -25,6 +35,8 @@ def cluster(request, ethermint, geth):
         yield ethermint
     elif provider == "geth":
         yield geth
+    elif provider == "pruned":
+        yield pruned
```

Comment:
For pruned node, the state is not pruned immediately, when the state is not pruned, it should work as a normal node, we only need to test the apis when the state is pruned, so maybe no need to run all the tests on it by default.